test/site/index_test.go

```diff
@@ -69,14 +69,14 @@ func TestHugoBundles(t *testing.T) {
 		if err != nil {
 			return err // unable to open
 		}
-		bundleFiles := []string{"_index.md", "index.md", "index.html", "_index.html"}
+		bundleFiles := []string{"index.md", "index.html", "_index.html"}
```

Comment:
@julz and also is this file redundant?

Reply:
We already skip this test (see line 34) nowadays, so it never runs. We could/should probably actually just delete the whole file at this point.